package.json

```
@@ -0,0 +1,30 @@
+{
+  "type": "module",
+  "name": "call_of_duty",
```

When you update the repo name, fix it here to `cabab-case`
```suggestion
  "name": "call-of-duty",
```

---

done